acceptance/samples/policy-input-golden-container.json

```diff
@@ -1037,7 +1038,8 @@
                   },
                   "parameters": {
                     "BASE_IMAGES": "",
-                    "BINARY_IMAGE": "quay.io/redhat-user-workloads/rhtap-contract-tenant/golden-container/golden-container:d11c0ada39f18f631ff4f54beafa72a9b62dcd7c"
+                    "BINARY_IMAGE": "quay.io/redhat-user-workloads/rhtap-contract-tenant/golden-container/golden-container:d11c0ada39f18f631ff4f54beafa72a9b62dcd7c",
+                    "HERMETIC": "true"
```

Comment:
I'm trying to understand how this is needed. In https://github.com/konflux-ci/build-definitions/blob/main/task/source-build-oci-ta/0.2/source-build-oci-ta.yaml I don't see a HERMETIC param. Am I looking in the right place?

Reply:
Does the acceptance test fail if this is removed?

Reply:
Not sure...